CI/publish_deps.ps1

```diff
@@ -9,7 +9,10 @@ cd $src_dir
 mkdir dependencies
 cd dependencies
 wget http://swdownloads.analog.com/cse/build/libiio-deps-20220517.zip -OutFile "libiio-win-deps.zip"
+wget https://swdownloads.analog.com/cse/build/libiio-deps-20240627.zip -OutFile "libiio-msvc-deps.zip"
```

Comment:
It would help (in the future) to rename libiio-deps-20200627 to a name that doesn't change over time. For example: libiio-deps-msvc. If at some point in time, there will be a new libiio-deps-yyyymmdd ,then there will be only one place to update the name. Now there are extra 4 places besides this one.

Reply:
i used the yyyy-mm-dd format, to match the naming used in the other archive. 
as a side note, the old archive with mingw compiled deps should also be updated to contain all required files. would it make sense to update the names of both archives to leave out the yyyy-mm-dd suffix after fixing the mingw archive as well ?